docbot/README.md

```diff
@@ -0,0 +1,54 @@
+# Documentation Bot 
+
+Automatically label pull requests based on the checked task list.
+
+## Usage
+
+Create a workflow `.github/workflows/ci-docbot.yml` with below content:
+
+```yaml
+name: Documentation Bot
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - labeled
+      - unlabeled
+
+jobs:
+  label:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout action
+        uses: actions/checkout@v3
+        with:
+          repository: maxsxu/action-labeler
```

Comment:
Did you meant to leave this reference to your personal github repo?

Reply:
I guess so. Any concern on this?

Reply:
My only concern is that the README is incorrect. We should fix it so that it documents how to use the action in this repository.

Reply:
Make sense! @maxsxu will update the readme.

Reply:
Fixed in https://github.com/apache/pulsar-test-infra/pull/48